### DISCUSSION TITLE
Vertical Scroll and Zoom

### DISCUSSION THREAD

**Discussion by Raheel-RDR** [igordanchenko/yet-another-react-lightbox#79] [General]
Hi I have been using yet-another-react-lightbox and it is great.  I have two things I want to ask. 

1. Is it possible to scroll vertical instead of horizontal? If so how? 
2. The zoom functionality seems to not be working using nextJS. I tried to add maxPixelZoom but it still did not work.

Thanks!

**Comment by igordanchenko** [comment]
Hi there!

1) yet-another-react-lightbox supports only horizontal scrolling
2) It's tough to guess what exactly you are doing wrong. Please share a code sandbox, and I'll take a look.

**Reply by Raheel-RDR** [reply]
Thanks for the quick reply. I have disabled next image for now and it is working. Once I am back and try the next Image rendering then I will try to post the code sandbox.

**Reply by igordanchenko** [reply]
You can use the following example as your starting point - https://codesandbox.io/p/sandbox/yet-another-react-lightbox-nextjs-bfjgb0?file=%2Fpages%2Findex.jsx&initialpath=%2F

Just adding Zoom plugin should do the trick. Let me know if you run into any issues.

**Reply by igordanchenko** [reply]
Here is my currently recommended approach of using Zoom plugin with Next.js image.

https://yet-another-react-lightbox.com/examples/nextjs#WithZoomPlugin
